packages/@adobe/spectrum-css-temp/components/modal/index.css

```diff
@@ -52,6 +52,11 @@ governing permissions and limitations under the License.
   &.is-open {
     visibility: visible;
   }
+
+  &.spectrum-fullscreenTakeover-Modal-wrapper {
+    /* Appear below nested modal underlays */
+    z-index: 1;
```

Comment:
below? usually setting a z-index goes above, can you point to exactly what class it's comparing with?

Reply:
So the case this handles is if you have a fullScreenTakeover modal and something in that modal triggers another modal dialog (as per a Spectrum example: https://spectrum-contributions.corp.adobe.com/page/takeover-dialog-beta/#Full-screen-takeover-variant). This can be tested via the new storybook story I added (`nested modals, fullscreentakeover`)

Since the z-index for normal modals is 2 (`spectrum-Modal-wrapper`), I made the fullscreenTakeoverModal wrapper have a z-index of 1 instead so the second modal + its underlay appears over the fullscreen takeover modal properly.

Reply:
hmm, I do not like this
also, what will happen when people put tooltips/popovers into these? will those all still work?
where will toasts fall?
We should probably add some z-index management to rsp and declare different z-index ranges (like radio wave frequency bands)
Just adding it to the CSS is going to get messy I think
Also, if we expose that, then people won't go crazy adding 1000000 z-index to things in their apps

Reply:
lets followup on it

Reply:
sure, created https://jira.corp.adobe.com/browse/RSP-1716 for followup